### PR TITLE
[PP-20] Fix error after fe deploy

### PR DESCRIPTION
**Pull request by Praddby**
Task: https://andrei-lysiuk.atlassian.net/browse/PP-19
Description:

  - remove "web: yarn start" in code;
  - add buildpack for heroku: https://github.com/mars/create-react-app-buildpack;
  - checked "web: bin/boot" in heroku.

### CODE REVIEW

File: src/untils/NavMap/NavMap.test.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { navMap } from "@/NavMap";
+import { navMap } from "@/untils/NavMap";
```
Comment (mrakofff) [inline src/untils/NavMap/NavMap.test.ts]: use relative path

File: src/untils/NavMap/NavMap.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class NavMap {
   constructor (private baseUrl = '') {}
 
-  home = () => `${this.baseUrl}`;
+  home = () => `${this.baseUrl}/`;
   lobby = () => `${this.baseUrl}/lobby/:id`;
```
Comment (mrakofff) [inline src/untils/NavMap/NavMap.ts]: update field with `lobbies`

File: src/untils/NavMap/NavMap.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   game = () => `${this.baseUrl}/game/:id`;
```
Comment (mrakofff) [inline src/untils/NavMap/NavMap.ts]: update field with `games`

**Review by mrakofff — APPROVED** [review]

File: src/untils/NavMap/NavMap.test.ts (hunk 1 above)
Comment (Praddby) [reply src/untils/NavMap/NavMap.test.ts]: Ok

File: src/untils/NavMap/NavMap.ts (hunk 2 above)
Comment (Praddby) [reply src/untils/NavMap/NavMap.ts]: ok

File: src/untils/NavMap/NavMap.ts (hunk 3 above)
Comment (Praddby) [reply src/untils/NavMap/NavMap.ts]: ok

**Pull request merged**